storops/unity/resource/user_quota.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright (c) 2015 EMC Corporation.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+from __future__ import unicode_literals
+
+import logging
+
+import storops
+from storops.unity.resource import UnityResource, UnityResourceList
+from storops.unity.client import UnityClient
+
+__author__ = 'Rajendra Indukuri'
+
+log = logging.getLogger(__name__)
+
+
+class UnityUserQuota(UnityResource):
+    """
+    Support CRUD operations for user_quota
+
+    Operations supported:
+    create: Create user_quota on a file_system/tree_quota
+    modify: Modify user_quota on a file_system/tree_quota
+    """
+    @classmethod
+    def create(cls, cli, file_system_id=None, tree_quota_id=None,
```

Comment:
It is better to use filesystem_id, we also consider filesystem as one word in Unity code base.